src/table/table.scss

```diff
@@ -122,265 +236,315 @@
         left: 0;
         font-size: $iui-font-size-title;
+        @include themed {
+          color: t(iui-color-foreground-positive);
+        }
       }
     }
 
-    &.iui-tables-row-disabled {
+    &.iui-disabled,
+    &.iui-disabled + .iui-expanded-content {
+      font-style: italic;
+      cursor: not-allowed;
       @include themed {
         color: t(iui-text-color-muted);
       }
-      font-style: italic;
-      cursor: not-allowed;
 
-      &:hover {
-        @include themed {
-          background-color: t(iui-color-background-1);
-        }
+      &:hover > .iui-slot > .iui-more-options {
+        visibility: hidden;
+      }
 
-        > .iui-tables-slot > .iui-tables-menu {
-          visibility: hidden;
-        }
+      img,
+      svg:not(.iui-radial),
+      .iui-user-icon {
+        filter: $iui-icons-color-multicolor-disabled;
       }
+    }
+
+    &.iui-positive {
+      @include iui-table-row-status;
+    }
+    
+    &.iui-warning {
+      @include iui-table-row-status($status:warning);
+    }
+
+    &.iui-negative {
+      @include iui-table-row-status($status:negative);
+    }
 
-      > .iui-tables-cell {
-        > img,
-        > svg,
-        > .iui-user-icon,
-        > .iui-user-icon > img {
-          filter: $iui-icons-color-multicolor-disabled;
+    &.iui-expanded {
+      .iui-expander > .iui-icon {
+        transform: rotate(90deg);
+      }
+
+      + .iui-expanded-content {
+        @include themed {
+          border-top: solid 1px t(iui-color-background-1);
         }
       }
     }
+    // #endregion Selection
   }
 
-  .iui-tables-message-container {
+  // Empty & loading states
+  &.iui-message {
+    text-align: center;
+    height: 100%;
+    padding: $iui-xl;
     @include themed {
       color: t(iui-text-color-muted);
       background-color: t(iui-color-background-2);
     }
+  }
+}
+
+@mixin iui-table-cell {
+  align-items: center;
+  display: flex;
+  flex-grow: 1;
+  min-width: $iui-l * 6;
+  min-height: $iui-baseline * 5;
+  padding: 0 $iui-m;
+  overflow: hidden;
+  flex-basis: $iui-l * 6;
+
+  > svg {
+    width: $iui-icons-default;
+    height: $iui-icons-default;
+    flex-shrink: 0;
+  }
+
+  > .iui-status,
+  > .iui-progress-indicator-radial {
+    float: right;
+    margin-left: auto;
+  }
+
+  > .iui-status {
+    margin-right: $iui-xs;
+  }
+
+  > .iui-user-icon {
+    margin-right: $iui-s;
+  }
+
+  > .iui-expander {
+    margin: 0 4px;
+  }
+
+  &.iui-slot {
+    width: $iui-l * 2;
+    padding: 0;
+    flex-grow: 0;
+    min-width: 0;
     display: flex;
     justify-content: center;
-    height: 100%;
-    padding: $iui-xl;
+    align-items: center;
+    flex-basis: $iui-l * 2;
+
+    > .iui-icon {
+      @include themed {
+        fill: t(iui-icons-color-actionable);
+        color: t(iui-icons-color-actionable);
+      }
+      width: $iui-icons-default;
+      height: $iui-icons-default;
+      margin: 0;
+    }
+  }
+    
+  &.iui-right-align {
```

Comment:
joking, right? :D

Reply:
Maybe? 👀

We want number columns (file size for example) to be right aligned for easier scanning by the user.  Would you rather this be defined in the HTML CSS?

Reply:
Gone 💥